### PR TITLE
Added Relational operator grammer in the csplat.y

### PR DESCRIPTION
**Pull request by fhzzzs**

### CODE REVIEW

File: csplat.y [hunk 1]
```diff
@@ -21,18 +21,31 @@ void yyerror (char const *err) { fprintf(stderr, "yyerror: %s\n", err); exit(-1)
 
 %%
 
-program: stmts {}
+program: stmts { printf("program -> stmts\n");}
+| program stmt { printf("program -> program stmt\n");}
 
-stmts: stmt {}
+stmts: add_exp ASSIGN { printf("stmts -> add_exp ASSIGN\n");}
```
Comment (nzimm012) [inline csplat.y:27]: How are these related to "statements"? Each of them are one statement aren't they?

File: csplat.y [hunk 2]
```diff
@@ -21,18 +21,31 @@ void yyerror (char const *err) { fprintf(stderr, "yyerror: %s\n", err); exit(-1)
 
 %%
 
-program: stmts {}
+program: stmts { printf("program -> stmts\n");}
+| program stmt { printf("program -> program stmt\n");}
```
Comment (nzimm012) [inline csplat.y:25]: We are not checking for semicolons with these rules are we?

**Review by nzimm012 — APPROVED** [review]
Some open questions but I would still merge the changes, so we can continue working on the missing rules.

Is the other merge request still relevant? I think this merge request is based on the other isn't it? If so, please close the other one.

File: csplat.y (hunk 1 above)
Comment (fhzzzs) [reply csplat.y:27]: idk, it's already named stmts when i get it

**Review by Steven-Eon — APPROVED** [review]
Looks good to me -- recommend merge to avoid merge conflicts with other branches later down the line.
